core/src/main/java/com/microsoft/applicationinsights/customExceptions/FriendlyException.java

```diff
@@ -0,0 +1,38 @@
+package com.microsoft.applicationinsights.customExceptions;
+
+public class FriendlyException extends Exception {
```

Comment:
can we try to remove this duplicate class?

Reply:
As of now we duplicated this class because we are not able to import core packages to agent-bootstrap. Added a note in one of the classes that the class is duplicated.